src/DssSpell.t.sol

```diff
@@ -614,369 +604,426 @@ contract DssSpellTest is DssSpellTestBase {
+        // assertEq(rwa015AInputConduitUrn.may(address(this)), 1);
 
         // transfer PSM GEM to input conduit
-        psmGem.transfer(address(rwaconduitinurn_014), pushAmount / daiPsmGemDiffDecimals);
-        assertEq(psmGem.balanceOf(address(rwaconduitinurn_014)), pushAmount / daiPsmGemDiffDecimals, "RWA014: Psm GEM not sent to input conduit");
-        
+        psmGem.transfer(address(rwa015AInputConduitUrn), pushAmount / daiPsmGemDiffDecimals);
+        assertEq(psmGem.balanceOf(address(rwa015AInputConduitUrn)), pushAmount / daiPsmGemDiffDecimals, "RWA015-A: Psm GEM not sent to input conduit");
+
         // input conduit 'push()' to the urn
-        rwaconduitinurn_014.push();
+        rwa015AInputConduitUrn.push();
 
-        assertEq(dai.balanceOf(address(rwaurn_014)), daiToPay, "Balance of the URN doesnt match");
+        assertEq(dai.balanceOf(address(rwa015AUrn)), daiToPay, "Balance of the URN doesnt match");
 
         // repay debt and free our collateral
-        rwaurn_014.wipe(daiToPay);
-        rwaurn_014.free(1 * WAD);
+        rwa015AUrn.wipe(daiToPay);
+        rwa015AUrn.free(1 * WAD);
 
-        // check if we get back RWA014 Tokens
-        assertEq(rwagem_014.balanceOf(address(this)), 1 * WAD, "RWA014: gem not sent back to the caller");
+        // check if we get back RWA015 Tokens
+        assertEq(rwa015AGem.balanceOf(address(this)), 1 * WAD, "RWA015-A: gem not sent back to the caller");
 
         // check if we have 0 collateral and outstanding debt in the VAT
-        (ink, art) = vat.urns("RWA014-A", address(rwaurn_014));
-        assertEq(ink, 0, "RWA014: bad `ink` after free()");
-        assertEq(art, 0, "RWA014: bad `art` after wipe()");
+        (ink, art) = vat.urns("RWA015-A", address(rwa015AUrn));
+        assertEq(ink, 0, "RWA015-A: bad `ink` after free()");
+        assertEq(art, 0, "RWA015-A: bad `art` after wipe()");
     }
 
-    function testFailRWA014_DRAW_ABOVE_LINE() public {
+    function testFailRWA015_A_DRAW_ABOVE_LINE() public {
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        uint256 drawAmount = 500_000_001 * WAD;
+        uint256 drawAmount = 2_500_001 * WAD;
 
         // setting address(this) as operator
-        vm.store(address(rwaurn_014), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
+        vm.store(address(rwa015AUrn), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
 
         // Draw 500m + 1
-        rwaurn_014.draw(drawAmount);
+        rwa015AUrn.draw(drawAmount);
     }
 
-    function testFailRWA014_OUTPUT_CONDUIT_PUSH_ABOVE_BALANCE() public {
+    function testFailRWA015_A_OUTPUT_CONDUIT_PUSH_ABOVE_BALANCE() public {
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        uint256 drawAmount = 500_000_000 * WAD;
+        uint256 drawAmount = 1_000 * WAD;
```

Comment:
Last spell we used `line` of RWA014 (500M).

This spell we are now using 1000 when line of 015 is 2.5M

Please use 2.5M as this value unless there is a reason to break precedent :)